docs/modules/operation/pages/deep-dive/meta-data.adoc

```diff
@@ -145,6 +145,12 @@ image::metadata/metadata_add.png["{page-component-title} UI displaying the Meta-
 . In the *Scope* list, select either *Node* or *Interface*.
 . Specify the key and a value, and click *Save*.
 
+[[ga-metadata-passwords]]
+== Hidden passwords in metadata
+
+Metadata's values of key names that includes a substring match of `password`, `Password`, `secret` or `Secret` will be hidden by default for non administrator users.
```

Comment:
```suggestion
Metadata values for key names that include a substring match of `password`, `Password`, `secret`, or `Secret` will be hidden by default for non-administrator users.
```
is it the key names or the values that include the substrings?

Reply:
@Bonrob2 The key names.